browser-test/src/admin_program_translation.test.ts

```diff
@@ -63,7 +63,7 @@ describe('Admin can manage translations', () => {
 
 
     // Expect program details link to contain 'Detalles del programa' with link to 'http://seattle.gov'
-    expect(await page.innerText('.cf-application-card a[href="http://seattle.gov"]')).toContain('Detalles del programa');
+    expect(await page.innerText('.cf-application-card a[href="http://seattle.gov"]')).toContain('External Site');

```

Comment:
question: Why did this change to english?

Reply:
Maybe we just need to add an "External site" Spanish translation to messages.es-us.

Reply:
Yeah, it's in English because the Spanish translation isn't available. Will update.